### PR TITLE
New welcome page and updated quick guide

### PR DESCRIPTION
**Pull request by PierrePaul**
Add a new welcome page to help Healthcare Workers get a feeling of the generate code page without pressing the "Generate key" button.

The welcome page is only shown once, after the signup, but before the 2fa.

https://trello.com/c/is3F7lex/321-new-staff-accounts-can-gain-familiarity-with-the-portal-without-generating-real-otks

Screenshot_2020-09-17 Welcome to the portal — COVID Alert Portal

Screenshot_2020-09-17 Welcome to the portal — Portail Alerte COVID

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Adds a new Welcome page to users who get through the invite + signup process.
```
Comment (pcraig3) [inline CHANGELOG.md:10]: Also updates the Quick Guide with more information

File: profiles/views.py [hunk 2]
```diff
@@ -125,7 +125,7 @@ def get_success_url(self):
 class SignUpView(FormView):
     form_class = SignupForm
     template_name = "profiles/signup.html"
-    success_url = reverse_lazy("login-2fa")
+    success_url = reverse_lazy("welcome")
```
Comment (pcraig3) [inline profiles/views.py:128]: Interesting. Going through it, it feels like this should come after the 2FA sign in.

**Review by pcraig3 — COMMENTED** [review]
Not the most important issue, but the warning message rolls over the code on smaller screens.

Easy fixes would be to add extra space to the top on smaller screens, or saying 50/50 width.

image

**Comment by pcraig3** [comment]
Opened a new PR to address the comments I made. If you approve, @PierrePaul, we can merge that one and close this one.

**Comment by PierrePaul** [comment]
Closing in favor of https://github.com/cds-snc/covid-alert-portal/pull/228

**Pull request closed without merging**